src/main/resources/default/taglib/t/heading.html.pasta

```diff
@@ -1,17 +1,17 @@
 <i:arg type="String" name="labelKey" default=""/>
 <i:arg type="String" name="label" default="@i18n(labelKey)"/>
-<i:arg type="String" name="class" default="" />
-<i:local name="markupLabel" value="@renderToString('body')" />
+<i:arg type="String" name="class" default=""/>
+<i:local name="markupLabel" value="@renderToString('body')"/>
 <i:if test="isFilled(label)">
     <div class="bb-gray pb-2 mt-4 mb-3 h5 text-black-75 align-items-end d-flex flex-row @class">
-        <span class="overflow-hidden">@label</span>
-        <span class="ml-auto"><i:render name="actions" /></span>
+        <span class="overflow-hidden">@label<i:render name="label"/></span>
```

Comment:
Do we really want to support both? In similar cases, we often seem to check for whether one or the other is filled and only print one.

Reply:
The use case is to place additional information next to a heading provided by a sirius lib via an extension in a product. This seemed to be the easiest way to achieve this.

Alternatively, I could also try and find another method to append things via JS.

Feel free to comment, which approach I should rather follow.

Reply:
That sounds legit. Then I am just wondering whether the block should be named `label-extension` instead.

In the straight-forward use case, I find it a bit strange to do the content as attribute and the extension as nested block in this case. But that is also a matter of taste, of course.

Reply:
(Add yes, I would hope that people would use the body version)

Reply:
commonly the i:block replaces the parameter. Check card.html.pasta - we ofter renderToString as fallback to the plain parameter..I'd keep it consistent here

Reply:
For the consistent case, there is already `markupLabel`. @cschierle's use case is truly remarkable 😅

Reply:
Where do we use this at all?